certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1952 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CognitiveServicesSpeechToText",
+    "description": "Azure Cognitive Services for Batch Speech-to-text",
+    "version": "1.0",
+    "contact": {
+      "name": "Cognitive Services - Speech Services",
+      "url": "https://docs.microsoft.com/azure/cognitive-services/speech-service/support"
+    }
+  },
+  "host": "eastus.api.cognitive.microsoft.com",
+  "basePath": "/speechtotext/v3.0",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/transcriptions": {
+      "get": {
+        "tags": [
+          "Custom Speech transcriptions:"
+        ],
+        "description": "Gets a list of transcriptions for the authenticated subscription.",
+        "operationId": "Transcriptions_List",
```

Comment:
@RichardSunMS all operationIds should be in Camel case. please change to "TranscriptionsList". Please fix all operationIds in your swagger file. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)